pep/app/pods/components/modal-dialogs/user/info/component.ts

```diff
@@ -1,12 +1,24 @@
 import CurrentUserService from 'pep/services/current-user';
 
 import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
 import Component from '@glimmer/component';
+import PepSessionService from 'pep/services/pep-session';
 
 interface ModalDialogsUserInfoArgs {
     onClose: () => void;
 }
 
 export default class ModalDialogsUserInfo extends Component<ModalDialogsUserInfoArgs> {
     @service currentUser!: CurrentUserService;
+    @service('pep-session') session!: PepSessionService;
+
+    /**
+     * Open the PaDS home-site using the user's generated
+     * link in a new tab.
+     */
+    @action
+    openPadsTab() {
+        window.open(this.currentUser.user!.paDSHomeURL, '_blank');
```

Comment:
@napafundi it would probably be safer her to wrap it inside an if block instead of using `!`

Reply:
I have the button wrapped in an `{{#if this.currentUser.user.paDSHomeURL}}`, that's why I was using the `!` here. Should I still add the if block here too?

Reply:
I would, in case you ever change it or try to use this method in another place that doesn't have that.